heron/config/src/yaml/BUILD

```diff
@@ -23,6 +23,11 @@ filegroup(
 )
 
 filegroup(
+    name = "conf-sandbox-yaml",
```

Comment:
who will use these files `conf-sandbox-yaml`

Reply:
@huijunw - this is for docker sandbox image.

Reply:
thanks @kramasamy